src/fix_press_berendsen.cpp

```diff
@@ -266,17 +266,22 @@ void FixPressBerendsen::init()
   // set temperature and pressure ptrs
 
   temperature = modify->get_compute_by_id(id_temp);
-  if (!temperature)
-    error->all(FLERR, "Temperature compute ID {} for fix press/berendsen does not exist", id_temp);
-
-  if (temperature->tempbias)
-    which = BIAS;
-  else
-    which = NOBIAS;
+  if (!temperature) {
+    error->all(FLERR, "Temperature compute ID {} for fix {} does not exist", id_temp, style);
+    if (temperature->tempflag == 0)
```

Comment:
Does fix press/berendsen even use the temperature? I see compute calls, but they look unnecessary.

Reply:
not sure what you mean - press/berend creates a pressure compute and passes it the temp compute it also created - then in end_of_step() it invokes both and uses the result in couple()

Reply:
Ah I missed that the id was passed to the pressure compute.